pages/c/_cluster/settings/brand.vue

```diff
@@ -27,10 +28,16 @@ export default {
       uiBannerSetting:    this.$store.dispatch('management/find', { type: MANAGEMENT.SETTING, id: SETTING.BANNERS }),
       uiLogoDarkSetting:  fetchOrCreateSetting(this.$store, SETTING.LOGO_DARK, ''),
       uiLogoLightSetting: fetchOrCreateSetting(this.$store, SETTING.LOGO_LIGHT, ''),
+      uiColorSetting:     fetchOrCreateSetting(this.$store, SETTING.PRIMARY_COLOR, ''),
+      hasSupportSetting:     fetchOrCreateSetting(this.$store, SETTING.SUPPORTED, ''),
     });
 
     Object.assign(this, hash);
 
+    if (this.hasSupportSetting.value === 'true') {
+      this.defaultColor = this.t('branding.color.defaultSuse');
```

Comment:
You could lose this defaultColor and just change the message to say 'You can override the primary color used throughout the UI with a custom color of your choice'